packages/components/src/range-control/index.js

```diff
@@ -64,9 +60,12 @@ function RangeControl( {
 		// The input is valid, reset the local state property used to temporaly save the value,
 		// and call onChange with the new value as a number.
 		resetCurrentInput();
-		onChange( newNumericValue );
+		onChange( ( newValue === undefined || newValue === '' ) ?
+			newValue :
```

Comment:
Do we want to pass the empty string to `onChange` ? Or should we be passing `undefined` always here?

Reply:
Nice catch, given that the value we components users receive is number it does not make any sense to pass an empty string. I updated the code to pass undefined.